packages/clarity/package.json

```diff
@@ -31,7 +31,8 @@
     "prepublishOnly": "npm run build",
     "lint": "tslint -p tsconfig.json 'src/**/*.ts' 'test/**/*.ts'",
     "lint-fix": "tslint --fix -p tsconfig.json 'src/**/*.ts' 'test/**/*.ts'",
-    "test": "npm run build && nyc mocha"
+    "test": "npm run build && nyc mocha",
+    "docs": "typedoc --out docs --plugin typedoc-plugin-markdown src"
```

Comment:
I see in the PR description "adds a generated docs.json" and a `docs.json` file committed. Does this command still output a json file or only markdown?

Reply:
The output is md in docs folder. The generation should be automated somehow and pushed to docs.

Reply:
maybe @blockstack-devops could help with that?

Reply:
In that case `packages/clarity/docs.json` should no longer be committed?

Reply:
has been removed now :+1:

Reply:
@CharlieC3 pinging you (instead of the team ping - not sure if that shows up for you). can we set up github actions for the generation?

Reply:
@agraebe Yup, would you mind creating an issue for it in the `devops` repo with details of what should be added?

Reply:
fyi - just created the issue for it